pkg/identity-manager/policy_handler.go

```diff
@@ -0,0 +1,87 @@
+///////////////////////////////////////////////////////////////////////
```

Comment:
This kind of confused me. The api-manager and image-manager defined this handler in the same controller.go file as the associated controller. We probably want to keep all of these consistent. I think the way you did it here makes more sense.

Reply:
Hmm, I kinda followed the suite from event manager. While working on this I noticed a lot of minor inconsistencies within different microservices. We need to fix them.